test/templates/dialog/index.njk

```diff
@@ -0,0 +1,209 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+{% from "./component.njk" import dialog %}
+{% from "../icon/component.njk" import icon %}
+{% set css_deps = ['fonts','components/button','icons','layout/overlay'] %}
+
+{% block content %}
+
+<h1>dialog</h1>
+
+
+{% set exampleSmall %}
+
+
+  {{  dialog(
+        properties={
+            header: "Modal dialog",
+            class: "fd-dialog__header",
+            size: "small",
+            body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.",
+            actions: [
+              {type: 'emphasized', label: 'Confirm'},
+              {type: 'secondary', label: 'Cancel'}
+            ]
+          }
+      )
+  }}
+
+{% endset %}
+
+{% set errorExampleSmall %}
+
+{{ dialog(
+        properties={
+          icon: "sap-icon--message-error",
+          header: "Error Message",
+          class: "fd-dialog__header-error",
+          size: "small",
+          body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.",
+          actions: [
+            {type: 'emphasized', label: 'Confirm'},
+            {type: 'secondary', label: 'Cancel'}
+          ]
+        }
+    ) 
+}}
+{% endset %}
+
+{% set warningExampleSmall %}
+
+{{ dialog(
+        properties={
+          icon: "sap-icon--message-warning",
```

Comment:
The icons should be part of the CSS and not passed in since these are specific to the type of warning. There is an `icons/mixin` that allows you to insert an icon in the CSS.